.github/workflows/ci.yml

```diff
@@ -42,8 +42,38 @@ jobs:
           use-mamba: true
           environment-file: environment.yml
           activate-environment: spox
-      - name: Run Unit Tests
+      - name: Run unit tests
         shell: bash -l {0}
         run: |
           pip install .
           pytest
+
+  opset-generation:
+    name: "Linux - opset-generation - Python ${{ matrix.PYTHON_VERSION }}"
+    runs-on: ubuntu-latest
+    env:
+      CI: True
+    strategy:
+      fail-fast: true
+      matrix:
+        PYTHON_VERSION: ['3.8']
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.PYTHON_VERSION }}
+          miniforge-variant: Mambaforge
+          miniforge-version: 4.11.0-2
+          use-mamba: true
+          environment-file: environment.yml
+          activate-environment: spox
+      - name: Run code generation
+        shell: bash -l {0}
+        run: |
+          pip install .
+          pip install pandoc
+          python codegen/generate.py
```

Comment:
Maybe we could add an `.old` suffix to the existing current modules here and then check if any of the files change? We're aiming for reproducibility, after all. Should just require some bash work (I assume the CI succeeds if there's just no non-zero status code?)

Reply:
Yes, currently it is only the exit code. I am not sure if checking for diffs is more trouble than it is worth. I am afraid that our CI will go red on every `onnx` release because some docstring was updated upstream. I guess we can just try it out and roll it back if need be.

Reply:
As we agreed, let's keep it simple for now and only test for errors.